app/js/components/LabOrdersList/LabOrdersList.jsx

```diff
@@ -7,13 +7,131 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React from 'react';
+import { connect } from 'react-redux';
+import cn from 'classnames';
+import moment from 'moment';
+import Loading from '../Loading';
+import SortableTable from '../SortableTable';
+import { fetchLabOrders } from '../../actions/labOrderAction';
+import "react-table/react-table.css";
+import "./index.scss";
+
+
+
+const Cell = ({columnName, value}) => {
+  if (columnName === "EMR ID") {
+    const emrID = value.patient.display.split('-')[0];
```

Comment:
We will create some utility methods for more deliberately selecting the primary identifier... but we haven't created these yet, so this is fine for now... can you just add a "TODO" above this to remind us? something like "TODO: refactor this and name column to use React Components patientUtils?)

Reply:
done!